system/library/razorpay/razorpay-sdk/.editorconfig

```diff
@@ -0,0 +1,14 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
@rohitcbr please remove this file.

Reply:
@yashgit891 removed in this commit https://github.com/razorpay/razorpay-opencart/pull/165/commits/94244e8ed3cc1cbb738076210f4125fe41409c8c